src/ontology/lexmatch/split-mapping-set/mondo_broadmatch_doid.tsv

```diff
@@ -1,3 +1,3 @@
 subject_id	subject_label	predicate_id	object_id	object_label	mapping_justification	mapping_tool	confidence	subject_match_field	object_match_field	match_string	comment
-MONDO:0015612	Dent disease	skos:broadMatch	DOID:0081453	Dent disease 1	semapv:LexicalMatching	oaklib	0.8	oio:hasNarrowSynonym	rdfs:label	dent disease 1	LEXMATCH
-MONDO:0015612	Dent disease	skos:broadMatch	DOID:0081454	Dent disease 2	semapv:LexicalMatching	oaklib	0.8	oio:hasNarrowSynonym	rdfs:label	dent disease 2	LEXMATCH
+MONDO:0015612	Dent disease	skos:broadMatch	DOID:0081453	Dent disease 1	semapv:LexicalMatching	oaklib	0.8	oio:hasNarrowSynonym	rdfs:label	dent disorder type 1	LEXMATCH
```

Comment:
## How common are label changes?
@twhetzel or @sabrinatoro I'm just reviewing this build, and I see that the label for `DOID:0081453` changed from `dent disease 1` to `dent disorder type 1`. There are also other labels that changed for other terms. Since I'm new to reviewing these outputs, I'm just wondering--are label changes pretty common?

Reply:
I think there is a dictionary of rules that adjusts the labels of terms in the external resource to try get better matches to Mondo labels.